### DISCUSSION TITLE
PoE or ModBus versions?

### DISCUSSION THREAD

**Discussion by twollweber** [nplan/HomeButtons#79] [Ideas]
Hi Nejc,

would it be an option to design a PoE version of the PCB? I noticed that the back cover of the enclosure features the 60 mm screw distance for European in-wall mounting. PoE would allow to provide permanent power and avoid the sometimes-not-so-good Wi-Fi. I have seen PoE ESP32 devices from Olimex and others but I have not investigated if there is room on your PCB layout to add the necessary components.

Thanks, Thorsten

**Comment by nplan** [comment]
Hi Thorsten,

I think it's possible. But it would most likely require an additional board to handle PoE. Mounted to the main PCB where the cell holder is currently.
